go/internal/multipeer-connectivity-transport/driver/mc-driver/mc-driver.h

```diff
@@ -0,0 +1,17 @@
+//
+//  mc-driver.h
+//  mc-driver
+//
+//  Created by Rémi BARBERO on 30/03/2020.
+//  Copyright © 2020 Rémi BARBERO. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+int StartMCDriver(char *localPId);
```

Comment:
You should return void here if the returns is ignored by libp2p transport

Reply:
done!